prowler/lib/outputs/finding.py

```diff
@@ -35,7 +35,7 @@ class Finding(BaseModel):
     status_extended: str
     muted: bool = False
     resource_uid: str
-    # resource_metadata: dict = Field(default_factory=dict) TODO: add resource_metadata to the finding
+    resource: dict = Field(default_factory=dict)
```

Comment:
Maybe here `resource_metadata` is more descriptive since this is the Prowler Finding.